napari/plugins/io.py

```diff
@@ -194,6 +203,31 @@ def save_layers(
     return written
 
 
+def _is_null_layer_sentinel(layer_data: Union[LayerData, Any]) -> bool:
+    """Checks if the layer data returned from a reader function indicates an
+    empty file. The sentinel value used for this is ``[(None,)]``.
+
+    Parameters
+    ----------
+    layer_data: LayerData
+        The layer data returned from a reader function to check
+
+    Returns
+    -------
+    bool
+        True, if the layer_data indicates an empty file, False otherwise
+    """
+    if not layer_data or not isinstance(layer_data, list):
+        return False
+    if len(layer_data) != 1:
+        return False
+    if not layer_data[0] or not isinstance(layer_data[0], tuple):
+        return False
+    if len(layer_data[0]) != 1:
+        return False
+    return layer_data[0][0] is None
```

Comment:
```suggestion
    return (
        isinstance(layer_data, list)
        and len(layer_data) == 1
        and isinstance(layer_data[0], tuple)
        and len(layer_data[0]) == 1
        and layer_data[0][0] is None
    )
```

Reply:
sorry to be so nitpicky... I just ran some perf tests and this is about 15% faster, and I think it reads a bit easier?

Reply:
Sure! Also like this version more, I just try to avoid multi-line statements as some people seem to really dislike them for whatever reason.